expan/core/experiment.py

```diff
@@ -141,3 +141,62 @@ def delta(self, method='fixed_horizon', **worker_args):
 
         result['kpis'] = kpis
         return result
+
+    @staticmethod
+    def _quant_apply(value, threshold):
+        if value < threshold:
+            return False
+        else:
+            return True
+
+    def _quantile_filtering(self, df, percentile):
```

Comment:
initially I didn't want to modify the existing data, but ultimately I decided to keep it simple... this is a leftover of the first approach, will fix